tools/tco/tco.cpp

```diff
@@ -0,0 +1,103 @@
+//===- tco.cpp - Tilikum Crossing Opt ---------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// =============================================================================
+//
+// This is to be like LLVM's opt program, only for FIR.  Such a program is
+// required for roundtrip testing, etc.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Conversion/LoopToStandard/ConvertLoopToStandard.h"
+#include "mlir/IR/MLIRContext.h"
+#include "mlir/IR/Module.h"
+#include "mlir/Parser.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassManager.h"
+#include "mlir/Transforms/Passes.h"
+#include "optimizer/CodeGen/CodeGen.h"
+#include "optimizer/FIRDialect.h"
+#include "optimizer/InternalNames.h"
+#include "optimizer/KindMapping.h"
+#include "optimizer/Transforms/Passes.h"
+#include "optimizer/Transforms/StdConverter.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/ErrorOr.h"
+#include "llvm/Support/InitLLVM.h"
+#include "llvm/Support/MemoryBuffer.h"
+#include "llvm/Support/SourceMgr.h"
+#include "llvm/Support/raw_ostream.h"
+
+using namespace llvm;
```

Comment:
I've notice that the first files you use llvm::, and by the end namespace started to be used.
Is there a reason for that? If not, why not use using namespace llvm for all?